consensus/hotstuff/signature.go

```diff
@@ -124,7 +124,7 @@ type Packer interface {
 	// sig is the aggregated signature data.
 	// Expected error returns during normal operations:
 	//  * none; all errors are symptoms of inconsistent input data or corrupted internal state.
-	Pack(blockID flow.Identifier, sig *BlockSignatureData) ([]flow.Identifier, []byte, error)
+	Pack(blockID flow.Identifier, sig *BlockSignatureData) ([]byte, []byte, error)
```

Comment:
```suggestion
	Pack(blockID flow.Identifier, sig *BlockSignatureData) (signerIndices []byte, signatures []byte, err error)
```
Could name the return values here for clarity - since they are the same type.

Reply:
How about we add a new type for bit vectors (defined under some common module maybe under `flow-go/module`) to differentiate bit vectors from `[]byte` ? 
The common module would provide the conversions to/from `[]flow.Identifier` and `BitVector`.

Reply:
Yeah, we can do that, [I've issued up,](https://github.com/onflow/flow-go/pull/2047#discussion_r822216975) will do it in a separate PR, since there is another PR already depend on this.